docs/UserGuide.md

```diff
@@ -282,10 +293,18 @@ Example csv:
 
 #### Exporting residents' emails : `export`
 
-Exports the emails of all the residents in the last filtered list to a filename of choice within the `data/exports/` folder.
-As they are comma-separated, a quick copy-paste allows you to send mass emails using modern email clients.
+Exports the emails of all the residents in the last filtered list to a csv file within the `data/exports/` folder.
+A quick copy-paste of the email addresses from the csv file allows you to send mass emails using modern email clients.
 
-Format: `export FILE_NAME`
+Format: `export CSV_NAME`
```

Comment:
This parameter name change will need to be reflected in the codebase. Also note command suggestion has a hardcoded  snippet for this as well.

Reply:
Good point, will leave it as FILE_NAME